docs/conf.py

```diff
@@ -48,7 +48,7 @@
 
 # General information about the project.
 project = u'Pillow (PIL Fork)'
-copyright = u'1995-2016, Fredrik Lundh and Contributors, Alex Clark and Contributors'
+copyright = u'1995-2017, Fredrik Lundh and Contributors, Alex Clark and Contributors'
```

Comment:
I agree with @aclark4life. Include the PIL copyright as it was, and add additional year ranges of other contributors, which at this point can be AC + Contributors.